crates/pathfinder/src/storage/schema/revision_0016.rs

```diff
@@ -15,3 +15,8 @@ pub(crate) fn migrate(transaction: &Transaction<'_>) -> anyhow::Result<()> {
 
     Ok(())
 }
+
+#[cfg(test)]
+mod tests {
+    // FIXME
+}
```

Comment:
How does a creating a table migration need tests?

Reply:
Also not present in the final state.